frontend/components/Campaign.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import browserHistory from 'react-router/lib/browserHistory';
+import NotFound from './NotFound.jsx';
+
+import Loading from './Loading.jsx';
+
+import utils from '../utils.js';
+import rootUrl from '../endpoint.js';
+
+
+class DonorRow extends Component {
+    render() {
+        return (
+            <div style={{ fontSize: "0.85rem", marginBottom: "0.5em" }}>
```

Comment:
this can be simplified to <li>$45 from {this.props.name}</li> the CSS should handle the proper styling already.